bot-twitter/README.md

```diff
@@ -16,11 +31,17 @@ $ npm run start
 
 ## Environment variables
 
+In order to configure you bot, you must create a `.env` file in the root of `/bot-twitter`.
+Below you can find a list of all available environment variables, along with an explanation of 
+what each variable determines. If not all parts of the bot are needed, you can leave some variables undefined.
```

Comment:
`.env` file is only one option of how to provide env variables. I would not include this text here since `Development Setup` already explains it